doc/ecosystem/index.rst

```diff
@@ -3,7 +3,14 @@
 Ecosystem
 =========
 
-Arbor exists in a wider computational neuroscience software ecosystem. The Arbor simulator is also not the only component written by the Arbor developers.
+Arbor exists in a wider computational neuroscience software ecosystem. In addition to the Arbor simulator, the Arbor developepment team writes and maintains a set of useful tools to be used in conjuction with Arbor.
```

Comment:
> developepment

Reply:
I guess there was a stray synapse encouraging me to write French, and the usual synapses generating noise f'ed it up.